### DISCUSSION TITLE
SAVE or ADD files to  sub-folder only

### DISCUSSION THREAD

**Discussion by Meche697** [rejetto/hfs#534] [Ideas]
Is there a way to set up folders such that those logged in can SAVE or ADD files to their sub-folder, but NOT download?  It's been a while and just starting to use this software again. I want to set up a system to share files where work has to be on their sub-folder and they can work exclusively off of the server ...

**Comment by rejetto** [comment]
set permissions like this
see: anyone
upload: anyone
downoad: noone

of course you may want to not use "anyone" but specific accounts or groups, that's up to you
